stdlib/pervasives.gr

```diff
@@ -91,18 +88,22 @@ export (++)
 
 // Checks the given items for structural equality.
 export (==)
-let (!=) : (a, a) -> Bool = (x, y) => !(x == y)
+let (!=): (a, a) -> Bool = (x, y) => !(x == y)
 // Checks the given items for physical equality.
 primitive (is) : (a, a) -> Bool = "@is"
 // The opposite of is operator
-let (isnt) : (a, a) -> Bool = (x, y) => !(x is y)
+let (isnt): (a, a) -> Bool = (x, y) => !(x is y)
 
 export enum List<a> { [], [...](a, List<a>) }
 
 /**
  * @deprecated This will be removed in a future release of Grain.
  */
-let cons = (a, b) => [a, ...b] // <- workaround for (grain-lang/grain#802) [TODO] fix #802 and delete
+let cons = (a, b) =>
+  [
+    a,
+    ...b
+  ] // <- workaround for (grain-lang/grain#802) [TODO] fix #802 and delete
```

Comment:
I think the comment caused this line to format weird

Reply:
I think it's a side effect of the formatting.  Even formatting it to
`let cons = (a, b) =>
  [a, ...b] // <- workaround for (grain-lang/grain#802) [TODO] fix #802 and delete
`
the comment line is over 80 char so needs to split again, which the formatter then does properly